src/FrontDoor/FrontDoor/Cmdlets/NewFrontDoorWafCustomRuleObject.cs

```diff
@@ -77,12 +78,17 @@ public class NewFrontDoorWafCustomRuleObject : AzureFrontDoorCmdletBase
         /// Enabled State. Possible values include: 'Enabled', 'Disabled'
         /// </summary>
         [Parameter(Mandatory = false, HelpMessage = "Enabled State. Possible values include: 'Enabled', 'Disabled'.")]
-        [PSArgumentCompleter("Enabled", "Disabled")]
         public string EnabledState { get; set; }
 
+        /// <summary>
+        /// Gets or sets describes the list of variables to group the rate limit
+        /// </summary>
+        [Parameter(Mandatory = false, HelpMessage = "Gets or sets describes the list of variables to group the rate limit")]
```

Comment:
As far as I can see, all the help messages for the new cmdlet and parameters need to be refined. They are publicly available to our customers, so we should make them more readable and understandable. The existing ones do not seem to fulfill this purpose. You could refer to the swagger for some explanations.